apps/dotcom/src/components/CursorChatBubble.tsx

```diff
@@ -126,11 +124,9 @@ const CursorChatInput = track(function CursorChatInput({
 	}, [editor, value, placeholder])
 
 	useLayoutEffect(() => {
-		// Focus the editor
-		let raf = requestAnimationFrame(() => {
-			raf = requestAnimationFrame(() => {
-				ref.current?.focus()
-			})
+		// Focus the input
+		const raf = requestAnimationFrame(() => {
```

Comment:
this double raf doesn't seem necessary... but maybe i'm missing something.

Reply:
yeah your version works in chrome, safari and firefox. cc @TodePond do you remember if this double raf was intentional?